rtc.go

```diff
@@ -109,11 +109,16 @@ type RTCConfig struct {
 	WebRTC WebRTCTransportConfig `mapstructure:"webrtc"`
 }
 
+type Signaller interface {
```

Comment:
Can you add the motivation to the documentation on the `Signaller` interface and indicate that this is a subset of `rtc.RTC_SignalClient`? The change looks fine, however it's a bit surprising because `ion-sdk-go` is pretty much only for `ion` so it's unexpected that the GRPC interface would not be sufficient. Some documentation would be good :)

Reply:
Hey Kevmo, thanks for the suggestions, I'll get back to them after a small break. 

To your other point, I found working directly with the ion-sfu was quite tricky as there were a lot of ion-isms to it. However the implementation seems solid for more applications than just the ion use case. One option would be to reverse engineer the ion-sdk / sfu to understand exactly what exactly is required to achieve a connection, and pray that neither implementation deviates too far to make maintenance impossible, or the alternative is to use the ion-sdk for it's desired use case - connecting to the ion-sfu - with a minor tweak. This way we can be sure that the two will continue to evolve in harmony. 

For my application the SFU is a subsection, not the main act, as it is with ion's video conferencing application, so you can imagine how daunting reimplementing a robust SFU would be. I'm hoping this small alteration might be able to help other in the future too.